tsconfig.json

```diff
@@ -16,10 +16,11 @@
       "es2015.collection",
       "es2015.promise",
       "dom"
-    ]
+    ],
+    "types": ["mocha", "node"]
```

Comment:
my other codebases does have types for these globals but didn't require this explicitly 🤔 any idea why it's needed?

Reply:
Probably because you were `declare`-ing things like `describe()` and `it()` somewhere else; but hey, this seems like the right thing to do and it fixes the errors so... yeah, LGTY?